_posts/2022-05-17-solidity-0.8.14-release-announcement.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+published: true
+title: 'Solidity 0.8.14 Release Announcement'
+date: '2022-03-17'
+author: Solidity Team
+category: Releases
+---
+
+[Solidity v0.8.14](https://github.com/ethereum/solidity/releases/tag/v0.8.14)
+fixes two important bugs.
+
+The first one is related to ABI-encoding nested arrays directly from calldata.
+You can find more information in the [blog post](/2022/05/17/calldata-reencode-size-check-bug/).
+
+The second bug is triggered in certain inheritance structures and can
+cause a memory pointer to be interpreted as a calldata pointer or
+vice-versa. We also have a dedicated [blog post](/2022/05/17/data-location-inheritance-bug/) about this bug.
+
+Apart from these, there are several minor bug fixes and improvements.
+
```

Comment:
```suggestion
Please note: Unfortunately, the npm package of this version is corrupted. Using the new ``soljson.js`` with solc-js 0.8.13 will work, but the (now marked as deprecated) 0.8.14 npm release is missing a file resulting in it not working properly. We released [0.8.14-fixed](https://www.npmjs.com/package/solc/v/0.8.14-fixed) in case you need a working npm package as dependency (Note however, that npm will not pick it up automatically).

```
Maybe something like this here?